src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2718" : " "); //if done return X
+    }
+    public void markAsDone(){
+        this.isDone = true;
+    }
+    public String toString() {
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+}
```

Comment:
I think that it would be good to standardize having a newline at the end of file for all files